src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java

```diff
@@ -518,6 +519,27 @@ public static void register() {
         tagProcessor.registerMechanism("name", false, ElementTag.class, (object, mechanism, param) -> {
             ((GuildChannel) object.getChannel()).getManager().setName(param.asString()).submit();
         });
+
+        // <--[mechanism]
+        // @object DiscordChannelTag
+        // @name topic
+        // @input ElementTag
+        // @description
+        // Changes the topic for this channel. The topic can only be 1024 characters or fewer.
+        // Does not work for thread or forum channels. Provide no input to reset the topic.
+        // -->
+        tagProcessor.registerMechanism("topic", false, (object, mechanism) -> {
+            if (!mechanism.hasValue()) {
+                ((TextChannel) object.getChannel()).getManager().setTopic(null).submit();

```

Comment:
Should probably cast to a `StandardGuildMessageChannel` here, so that it works for news channels as well